Marlin/src/lcd/language/language_it.h

```diff
@@ -346,14 +346,14 @@ namespace Language_it {
   PROGMEM Language_Str MSG_VMIN                            = _UxGT("Vmin");
   PROGMEM Language_Str MSG_VTRAV_MIN                       = _UxGT("VTrav min");
   PROGMEM Language_Str MSG_ACCELERATION                    = _UxGT("Accelerazione");
-  PROGMEM Language_Str MSG_AMAX_A                          = _UxGT("Amax ") LCD_STR_A;
-  PROGMEM Language_Str MSG_AMAX_B                          = _UxGT("Amax ") LCD_STR_B;
-  PROGMEM Language_Str MSG_AMAX_C                          = _UxGT("Amax ") LCD_STR_C;
-  PROGMEM Language_Str MSG_AMAX_I                          = _UxGT("Amax ") LCD_STR_I;
-  PROGMEM Language_Str MSG_AMAX_J                          = _UxGT("Amax ") LCD_STR_J;
-  PROGMEM Language_Str MSG_AMAX_K                          = _UxGT("Amax ") LCD_STR_K;
-  PROGMEM Language_Str MSG_AMAX_E                          = _UxGT("Amax ") LCD_STR_E;
-  PROGMEM Language_Str MSG_AMAX_EN                         = _UxGT("Amax *");
+  PROGMEM Language_Str MSG_AMAX_A                          = _UxGT("Max ") LCD_STR_A _UxGT("Accel");
```

Comment:
I suspect it should look like (added space in accel text):
```c++
_UxGT("Max` ") LCD_STR_A _UxGT(" Accel");
```

Reply:
Thanks! A little `LCD_STR_I _UxGT\("[^ -]` search helped find the rest.